pom.xml

```diff
@@ -162,7 +162,8 @@ limitations under the License.
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-shade-plugin</artifactId>
-          <version>3.3.0</version>
+          <!-- MSHADE-419: 3.3.0 Shade plugin causes pom to be created without compile dependencies -->
+          <version>3.2.4</version>
```

Comment:
please update the renovate config to ignore this so we don't accidentally bump it again